kiwmi/luak/kiwmi_keyboard.c

```diff
@@ -16,6 +16,29 @@
 #include "input/keyboard.h"
 #include "luak/kiwmi_lua_callback.h"
 
+static int
+l_kiwmi_keyboard_configure(lua_State *L)
+{
+    struct kiwmi_keyboard *keyboard =
```

Comment:
The item at index 1 is a `struct kiwmi_object *`, check how `keyboard:modifiers` accesses the keyboard.

Reply:
Oh yeah I developed it against an earlier commit where this wasn't wrapped this way. I could fix that